### PR TITLE
Adds wordpress detection

### PR DESCRIPTION
**Pull request by housseindjirdeh**
CC @wardpeet @paulirish

### CODE REVIEW

File: library/libraries.js [hunk 1]
```diff
@@ -1361,6 +1361,21 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
...
         }
     },
+    'WordPress': {
+        icon: 'wordpress',
+        url: 'https://wordpress.org/',
+        npm: null,
+        test: function (win) {
+            if (!!document.querySelector('meta[name=generator][content^="WordPress"]')) {
+                var versionName = document.querySelector("meta[name='generator']").getAttribute("content");
+                return { version: versionName.replace(/[^0-9\.]+/g,"") };
```
Comment (paulirish) [inline library/libraries.js]: over on https://wordpress.org/news/ i spotted `WordPress 5.2-alpha-44796` so I think we'll need something different..  
`.replace(/^\w+\s/,'')` perhaps?

Comment (westonruter) [reply library/libraries.js]: That's right. The version can look like any one of:

* `5.1.2`
* `5.1-beta2`
* `5.0-RC1`
* `5.2-alpha-44742-src`

So yes, just removing `'WordPress '` from the beginning will be the way to go.

Comment (housseindjirdeh) [reply library/libraries.js]: Ooh didn't know more than just numbered releases could be included. Fixed 👍

**Comment by housseindjirdeh** [comment]
@paulirish Updated it to your suggestion. I think it looks good.

One thing however: I used `if (!hasAPILinkElem && !hasWPIncludes)` instead of `||`. I'm assuming that we would want to fallback to `wp-includes` for WordPress versions older than 4.4, but somebody correct me if I'm wrong.

CC @Shelob9

**Comment by Shelob9** [comment]
@housseindjirdeh This looks good to me.

**Comment by westonruter** [comment]
Note: As of WordPress 6.3.0, sites will [no longer be outputting](https://core.trac.wordpress.org/ticket/41404) this `link`:

```html
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="https://example.com/wp-includes/wlwmanifest.xml">
```

The logic needs to be updated to not depend on it. See https://github.com/johnmichel/Library-Detector-for-Chrome/issues/203.

**Pull request merged**